Core/ALife.Core/ImportExport/AgentCodeSerializer.cs

```diff
@@ -30,7 +30,7 @@ public static string ExportAgentDefinitionAsCode(Agent theAgent, string name)
 
             StringBuilder outputCode = new StringBuilder();
             outputCode.AppendLine("int agentRadius = 5;");
-            outputCode.AppendLine("ApplyCircleShapeToAgent(parentZone.Distributor, System.Drawing.Color.Blue, agentRadius, 0);");
+            outputCode.AppendLine("ApplyCircleShapeToAgent(parentZone.Distributor, System.Drawing.Colour.Blue, agentRadius, 0);");
```

Comment:
This is incorrect. This is a string for how to "create" an object.

Reply:
I tried to not touch strings/comments, but looks like this got partially hit. I'll remove the System.Drawing part once I get home so this should function again